### ISSUE TITLE
1Password Alfred workflow 

### ISSUE THREAD

**Issue by clarker77** [alfredapp/1password-workflow#32]
### Workflow version
2022.13

### Alfred version
5.0 EA4

### macOS version
12.4

### Architecture
i386

### Workflow environment variables
Dict {
    auto_refresh = 1
    logins_only = 0
    hostnames_only = 1
}

### 1Password version
8.7.3

### op version
In Workflow: 2.2.0

### Debugger output

```codeblock
[09:12:26.504] Logging Started...
```

### Terminal output

```terminal_text
-2 user.workflow.426FBCE8-53BF-4224-A1E0-87046D9EC159 %
```

### More details

Try the 1Password workflow did not see what I expected to see namely a place to enter some of my 1Password credentials

**Comment by vitorgalvao** [comment]
There’s nothing obviously wrong in the diagnostic. Please provide [a video](https://support.apple.com/en-us/HT208721) of the process as you’re doing it.

**Comment by clarker77** [comment]
https://cvws.icloud-content.com/B/AbQ7WlOfk8eD24YzF5A__3ye6W0gAaeueUzA-guMNuzz3tEv4JDZBTRP/Screen+Recording+2022-06-29+at+11.07.01+AM.mov?o=AuMJiIdt6aqYZ7FojluiG8_IsXDdbuiCF3FQwnuER1nn&v=1&x=3&a=CAogwUdbwPPnmR1C3nQ-fP_lisMvPfPs1bOtxcyImbXOae0SbRDzrdKAmzAY84qugpswIgEAUgSe6W0gWgTZBTRPaiZVB5ZnbK0jojTIyeB0RrCYG1A2xzKWy37UTEmGLmdDY0NE49bVGXImI4V08imbQ36M8nBE9zmOR6LoHSE7kR4rOik0ir0AXzd4rxQHdNs&e=1656520148&fl=&r=e1105347-a13a-4e08-bd89-4dc99559265e-1&k=q9mKeZThFlY1q7WmTz1Hkw&ckc=com.apple.clouddocs&ckz=com.apple.CloudDocs&p=104&s=eA8Q06_s4pQmvlKXNo8ft9C2h3I&cd=i

**Comment by vitorgalvao** [comment]
The link isn’t working. It just says `Gone`.

**Comment by vitorgalvao** [comment]
No, you call Alfred’s search bar and type `1p`. Then you’ll see “Sign in and update items”. Have you watched [the video](https://twitter.com/vhgalvao/status/1542121499414380546)?

**Comment by clarker77** [comment]
Sorry to be a pain. I’m not seeing the Sign In and Update option that’s shown at the start of the video although I did see it a few days ago on one of my earlier attempts. In the video I don’t see the return symbol at the end of the line (which I do see). I may still be on the wrong screen?

**Comment by clarker77** [comment]
I was able to get to the proper screen and my only remaining problem seems to be with my 1password credentials. So I’ll take it from here. Thanks for your help.
